doc/guide/plugin.rst

```diff
@@ -590,6 +590,9 @@ shapes for its calculation, so the volume fraction parameter in the model
 is automatically scaled by *form_volume/shell_volume* prior to calling the
 structure factor.
 
+Special functions for scattering such as $3j_1(x)/x$ are available. See
+:mod:`sasmodels.special` for details.
+
 Embedded C Models
```

Comment:
It would really help to at least point out these functions, using the exact same name, are also provided for c models as described. In other words there is an extra incentive to use them.

However the link seems to point to the sasmodels developer docs rather than that section in this document even though they clearly both originate from the exact same rst file. I assume this is because the special functions section is listed UNDER c models rather than a section unto itself? 

If so I think the right answer should be to either pull out that section as a separate section and point out these are available both to python models and to  models (my preference) OR link to the same page in both places (using `:mod:sasmodels.special`)